test/files/neg/t11607.check

```diff
@@ -1,4 +1,4 @@
-t11607.scala:2: error: class migration in package annotation cannot be accessed as a member of package annotation from package <empty>
+t11607.scala:2: error: not found: type migration
```

Comment:
Compiler could use githubesque, "We went looking for that symbol everywhere but couldn't find it."